frontend/src/toolbar/toolbarLogic.ts

```diff
@@ -65,8 +65,8 @@ export const toolbarLogic = kea<toolbarLogicType>([
         },
         processUserIntent: async () => {
             if (props.userIntent === 'add-action' || props.userIntent === 'edit-action') {
```

Comment:
Yeah, I don't know how you would launch the toolbar and edit an action at the same time. I vote to delete it and then add it back if someone can tell us what broke